src/renderer/experimental.js

```diff
@@ -58,6 +58,14 @@ export const experimentalFeatures: Feature[] = [
     valueOn: "https://api.trongrid.io",
     valueOff: "https://tron.coin.ledger.com",
   },
+  {
+    type: "toggle",
+    name: "ELROND_API_ENDPOINT",
+    title: <Trans i18nKey="settings.experimental.features.apiElrondEndpPoint.title" />,
+    description: <Trans i18nKey="settings.experimental.features.apiElrondEndpPoint.description" />,
+    valueOn: "https://elrond.coin.ledger.com/ ",
+    valueOff: "https://api.elrond.com",
+  },
```

Comment:
checking with @FabriceDautriat , i think we want no experimental to be visible for end user until it's "go prod", so i think we want this commented or removed for now, so we can merge the PR in silent mode.